demos/vc_issuer/tests/issue_credential.rs

```diff
@@ -163,36 +163,41 @@ mod api {
 /// Verifies that the consent message can be requested.
 #[test]
 fn should_return_vc_consent_message() {
+    let test_cases = [
+        ("en-US", "en", "# DFINITY Foundation Employment Credential"),
+        ("de-DE", "de", "# Beschäftigungsausweis DFINITY Stiftung"),
+        ("ja_JP", "en", "# DFINITY Foundation Employment Credential"), // test fallback language
```

Comment:
not sure if meant to be so but you have a mix of xx-XX and xx_XX

Reply:
I didn't, thanks for pointing it out!